.pylintrc

```diff
@@ -3,7 +3,7 @@
 variable-rgx=[a-z0-9_]{1,30}$
 good-names=log
 
-disable=invalid-name,unused-argument,too-few-public-methods,no-self-use,missing-docstring,logging-format-interpolation,too-many-instance-attributes,duplicate-code,too-many-public-methods,too-many-arguments,protected-access,pointless-string-statement,too-many-lines,useless-object-inheritance,cyclic-import,bad-option-value
+disable=invalid-name,unused-argument,too-few-public-methods,no-self-use,missing-docstring,logging-format-interpolation,too-many-instance-attributes,duplicate-code,too-many-public-methods,too-many-arguments,protected-access,pointless-string-statement,too-many-lines,useless-object-inheritance,cyclic-import,bad-option-value,fixme
```

Comment:
Ok, done. I figured that because you had it in the github workflow, this wouldn't matter. I have an idea about why you don't want this, but could you please elaborate so I can better myself?

Thanks

Reply:
They are reminders. You should see them when running locally, but continuous tests shouldn't fail because of them. It's just my personal convention.